proto/cometbft/types/v1/params.proto

```diff
@@ -81,39 +88,45 @@ message HashedParams {
   int64 block_max_gas   = 2;
 }
 
-// SynchronyParams configure the bounds under which a proposed block's timestamp is considered valid.
-// These parameters are part of the proposer-based timestamps algorithm. For more information,
-// see the specification of proposer-based timestamps:
-// https://github.com/cometbft/cometbft/tree/main/spec/consensus/proposer-based-timestamp
+// SynchronyParams determine the validity of block timestamps.
+// 
+// These parameters are part of the Proposer-Based Timestamps (PBTS) algorithm.
+// For more information on the relationship of the synchrony parameters to
+// block timestamps validity, refer to the PBTS specification:
 message SynchronyParams {
-  // message_delay bounds how long a proposal message may take to reach all validators on a network
-  // and still be considered valid.
-  google.protobuf.Duration message_delay = 1
+  // Bound for how skewed a proposer's clock may be from any validator on the
+  // network while still producing valid proposals.
```

Comment:
```suggestion
  // network while still producing valid proposals, in ms.
```

Reply:
It is not. It is a `time.Duration`, an `int64`, given in nanos: https://pkg.go.dev/time#Duration.

If it said otherwise in other place, please let me known.